### PR TITLE
feat(azure-iot-hub): update swagger

### PR DESCRIPTION
**Pull request by zolvarga**
# Checklist
- [ ] I have read the [contribution guidelines] (https://github.com/Azure/azure-iot-sdk-node/blob/master/.github/CONTRIBUTING.md).
- [ ] I added or modified the existing tests to cover the change (we do not allow our test coverage to go down).
- If this is a modification that impacts the behavior of a public API
  - [ ] I edited the corresponding document in the `devdoc` folder and added or modified requirements.

# Reference/Link to the issue solved with this PR (if any)

# Description of the problem

# Description of the solution

### CODE REVIEW

File: digitaltwins/samples/service/javascript/get_digital_twin.js [hunk 1]
```diff
@@ -3,32 +3,31 @@
 
 const IoTHubTokenCredentials = require('azure-iot-digitaltwins-service').IoTHubTokenCredentials;
 const DigitalTwinServiceClient = require('azure-iot-digitaltwins-service').DigitalTwinServiceClient;
-
-const connectionString = process.env.IOTHUB_CONNECTION_STRING;
-const deviceId = process.env.IOTHUB_DEVICE_ID;
+const { inspect } = require('util');
```
Comment (anthonyvercolano) [inline digitaltwins/samples/service/javascript/get_digital_twin.js:6]: >inspect [](start = 8, length = 7)

Learn something new everyday.

File: digitaltwins/samples/service/javascript/get_model.js [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+// - Twin enabled device must exist on the ADT hub
+async function main() {
+  const modelId = process.env.IOTHUB_MODEL_ID; // suggestion: urn:azureiot:Client:SDKInformation:1
+
+  // Create service client
+  const credentials = new IoTHubTokenCredentials(process.env.IOTHUB_CONNECTION_STRING);
+  const digitalTwinServiceClient = new DigitalTwinServiceClient(credentials);
+
+  // Get digital twin model
+  const model = await digitalTwinServiceClient.getModel(modelId);
```
Comment (anthonyvercolano) [inline digitaltwins/samples/service/javascript/get_model.js:23]: >getModel(modelId) [](start = 47, length = 17)

Interesting that these can existing independent of the context of a device.

Is there an API to push a model into a hub?

**Comment by anthonyvercolano** [comment]
Where's the CL?

File: digitaltwins/samples/service/javascript/update_component.js [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+      properties: {
+        'brightness': { // for the environmental sensor, try "brightness"
+          desired: {
+            value: 42 // for the environmental sensor, try 42 (note that this is a number, not a string, so don't include quotes).
+          }
+        }
+      }
+    }
+  }
+}];
```
Comment (anthonyvercolano) [inline digitaltwins/samples/service/javascript/update_component.js:20]: So I'm presuming that the CL layer transforms this into the JSON patch description?

Comment (anthonyvercolano) [reply digitaltwins/samples/service/javascript/update_component.js:20]: Clearly I presume wrong.

---
In reply to: [450407004](https://github.com/Azure/azure-iot-sdk-node/pull/837#discussion_r450407004) [](ancestors = 450407004)

File: digitaltwins/samples/service/javascript/get_digital_twin.js (hunk 1 above)
Comment (zolvarga) [reply digitaltwins/samples/service/javascript/get_digital_twin.js:6]: Same here :)

---
In reply to: [450402230](https://github.com/Azure/azure-iot-sdk-node/pull/837#discussion_r450402230) [](ancestors = 450402230)

File: digitaltwins/samples/service/javascript/get_model.js (hunk 2 above)
Comment (zolvarga) [reply digitaltwins/samples/service/javascript/get_model.js:23]: No, there is no such API on IoTHub

---
In reply to: [450403979](https://github.com/Azure/azure-iot-sdk-node/pull/837#discussion_r450403979) [](ancestors = 450403979)

File: digitaltwins/samples/service/javascript/update_component.js (hunk 3 above)
Comment (zolvarga) [reply digitaltwins/samples/service/javascript/update_component.js:20]: No, it does not

---
In reply to: [450411859](https://github.com/Azure/azure-iot-sdk-node/pull/837#discussion_r450411859) [](ancestors = 450411859,450407004)

**Pull request merged**